testsrc/test262.properties

```diff
@@ -1407,47 +1403,25 @@ built-ins/RegExp 897/1464 (61.27%)
     prototype/flags/coercion-sticky.js
     prototype/flags/coercion-unicode.js
     prototype/flags/get-order.js {unsupported: [regexp-dotall]}
-    prototype/flags/length.js
-    prototype/flags/name.js
-    prototype/flags/prop-desc.js
     prototype/flags/rethrow.js {unsupported: [regexp-dotall]}
     prototype/flags/return-order.js {unsupported: [regexp-dotall]}
     prototype/flags/this-val-regexp.js {unsupported: [regexp-dotall]}
-    prototype/flags/this-val-regexp-prototype.js
-    prototype/global/15.10.7.2-2.js
     prototype/global/cross-realm.js {unsupported: [cross-realm]}
-    prototype/global/length.js
-    prototype/global/name.js
-    prototype/global/S15.10.7.2_A9.js
     prototype/global/this-val-regexp-prototype.js
-    prototype/ignoreCase/15.10.7.3-2.js
     prototype/ignoreCase/cross-realm.js {unsupported: [cross-realm]}
-    prototype/ignoreCase/length.js
-    prototype/ignoreCase/name.js
-    prototype/ignoreCase/S15.10.7.3_A9.js
     prototype/ignoreCase/this-val-regexp-prototype.js
-    prototype/multiline/15.10.7.4-2.js
     prototype/multiline/cross-realm.js {unsupported: [cross-realm]}
-    prototype/multiline/length.js
-    prototype/multiline/name.js
-    prototype/multiline/S15.10.7.4_A9.js
     prototype/multiline/this-val-regexp-prototype.js
     prototype/source/cross-realm.js {unsupported: [cross-realm]}
-    prototype/source/length.js
-    prototype/source/name.js
     prototype/source/prop-desc.js
     prototype/source/this-val-regexp-prototype.js
     prototype/source/value-empty.js
     prototype/source/value-line-terminator.js
     prototype/source/value-u.js
     prototype/sticky/cross-realm.js {unsupported: [cross-realm]}
-    prototype/sticky/length.js
-    prototype/sticky/name.js
-    prototype/sticky/prop-desc.js
     prototype/sticky/this-val-regexp-prototype.js
```

Comment:
This test is dependant on the sticky property being implemented as an accessor property. However, it looks like your changes haven't caused it to pass yet.

Not sure if that is due to the accessor property impl or if something else, possibly unrelated, is going on